### PR TITLE
Feature/ab#1276 Start with demo data

### PR DESCRIPTION
**Pull request by ztarbug**
Cockpit can now be started such that a set of demo data is imported. 

## Description
Via application properties a folder with sample files can be configured. Files in there will be imported to AICockpit's database.

## Motivation and Context
Cockpit can be demonstrated with various scenarios.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.

### CODE REVIEW

File: service/src/main/java/de/starwit/service/impl/PostFlywayService.java [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    private String scenarioImportFolder;
+
+    private String mititgationTypeFileName = "mitigationtypes.json";
+
+    private String incidentTypeFileName = "incidenttypes.json";
+
+    private String demoDataFileName = "demodata.json";
+
+    @Autowired
+    private TrafficIncidentTypeRepository ttRepository;
```
Comment (witchpou) [inline service/src/main/java/de/starwit/service/impl/PostFlywayService.java:51]: please use better names for repositories

File: service/src/main/java/de/starwit/service/impl/PostFlywayService.java [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+                    importDemoData(scenarioImportFolder);
+                }
+            }
+        }
+    }
+
+    private boolean importMitigationTypes(String folder) {
+        if (mtRepository.findAll().size() > 0) {
+            LOG.info("Mitigation types already imported. Skipping import.");
+        } else {
```
Comment (witchpou) [inline service/src/main/java/de/starwit/service/impl/PostFlywayService.java:78]: I would avoid else and return immediately

File: service/src/main/java/de/starwit/service/impl/PostFlywayService.java [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+                }
+                return true;
+            } else {
+                LOG.warn("Incident types file not found. Skipping import.");
+            }
+        }
+        return false;
+    }
+
+    private boolean importDemoData(String folder) {
```
Comment (witchpou) [inline service/src/main/java/de/starwit/service/impl/PostFlywayService.java:121]: The actions are not created correctly. I would use TrafficIncidentService and create a new method for import or adjust the create method

**Review by witchpou — CHANGES_REQUESTED** [review]

File: service/src/main/java/de/starwit/service/impl/PostFlywayService.java [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+        File file = new File(folder + "/" + demoDataFileName);
+        if (file.exists()) {
+            LOG.info("Importing demo data from file " + file.getAbsolutePath());
+            try {
+                Path path = Path.of(folder + "/" + demoDataFileName);
+                String content = Files.readString(path, StandardCharsets.UTF_8);
+                int timeOffset = 13;
```
Comment (witchpou) [inline service/src/main/java/de/starwit/service/impl/PostFlywayService.java:128]: I don't understand the handling of the timeOffset

File: service/src/main/java/de/starwit/service/impl/PostFlywayService.java (hunk 1 above)
Comment (ztarbug) [reply service/src/main/java/de/starwit/service/impl/PostFlywayService.java:51]: done

File: service/src/main/java/de/starwit/service/impl/PostFlywayService.java (hunk 2 above)
Comment (ztarbug) [reply service/src/main/java/de/starwit/service/impl/PostFlywayService.java:78]: done

File: service/src/main/java/de/starwit/service/impl/PostFlywayService.java (hunk 4 above)
Comment (ztarbug) [reply service/src/main/java/de/starwit/service/impl/PostFlywayService.java:128]: added comment

File: service/src/main/java/de/starwit/service/impl/PostFlywayService.java (hunk 3 above)
Comment (ztarbug) [reply service/src/main/java/de/starwit/service/impl/PostFlywayService.java:123]: won't fix here

**Pull request merged**